astro.config.mjs

```diff
@@ -24,7 +24,7 @@ export default defineConfig({
       title: "Alumnium",
       social: {
         github: "https://github.com/alumnium-hq/alumnium",
-        discord: "https://discord.gg/45hYBf3U",
```

Comment:
the link didn't work for me 🤷‍♀️

Reply:
Can you use https://discord.gg/mP29tTtKHg instead? It's a new invite link that never expires.
<img width="695" alt="image" src="https://github.com/user-attachments/assets/8e9ea0c4-3b61-4974-9a0c-1fbd95cec6e7" />